src/modules/health/indicators/dynamic-config.health.ts

```
@@ -0,0 +1,50 @@
+import { DynamicConfigService } from '@/modules/dynamic-config/services/dynamic-config.service';
+
+@Injectable()
+export class DynamicConfigHealthIndicator extends HealthIndicator {
+  constructor(private readonly dynamicConfigService: DynamicConfigService) {
+    super();
+  }
+
+  async isHealthy(key: string): Promise<HealthIndicatorResult> {
+    try {
+      const healthCheck = await this.dynamicConfigService.getDetailedHealthCheck();
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)